src/compressed_tensors/compressors/model_compressors/model_compressor.py

```diff
@@ -104,6 +104,8 @@ def from_pretrained(
         """
         config = AutoConfig.from_pretrained(pretrained_model_name_or_path, **kwargs)
         compression_config = getattr(config, COMPRESSION_CONFIG_NAME, None)
+        if compression_config is None:
+            compression_config = getattr(config, QUANTIZATION_CONFIG_NAME, None)
```

Comment:
Not required

Reply:
https://github.com/neuralmagic/compressed-tensors/pull/173